services/s3/types.go

```diff
@@ -279,3 +280,37 @@ type DeleteObjectsError struct {
 	Message string
 	//VersionId string
 }
+
+type ListObjectsV2Input struct {
+	Bucket            string  `s3:"bucket"`
+	ContinuationToken *string `s3:"query:continuation-token"`
+	MaxKeys           *int    `s3:"query:max-keys"`
+	Prefix            *string `s3:"query:prefix"`
```

Comment:
tbh I've been a bit lazy about nullability on these, but what you did is probably a little more correct